From 2da01758fe8fd9d1674362af49654a70df874a9d Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Mon, 4 Dec 2017 15:21:03 +0800
Subject: [PATCH 2973/5242] MLK-17072-2: ARM64: dts: freescale: imx8qm/qxp:
 enable MU as wakeup source

commit  e63ac87e302bcf90ca1faec10309b7c8fbb2acfc from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable MU as wakeup source in dts.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi  |    1 +
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index cdfe410..9819f00 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -104,6 +104,7 @@
 		compatible = "fsl,imx8-mu";
 		reg = <0x0 0x5d1b0000 0x0 0x10000>;
 		interrupts = <GIC_SPI 176 IRQ_TYPE_LEVEL_HIGH>;
+		interrupt-parent = <&wu>;
 		fsl,scu_ap_mu_id = <0>;
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 050f65a..9880c11 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -104,6 +104,7 @@
 		compatible = "fsl,imx8-mu";
 		reg = <0x0 0x5d1b0000 0x0 0x10000>;
 		interrupts = <GIC_SPI 176 IRQ_TYPE_LEVEL_HIGH>;
+		interrupt-parent = <&wu>;
 		fsl,scu_ap_mu_id = <0>;
 		status = "okay";
 	};
-- 
1.7.9.5

